lib/utm_form.coffee

```diff
@@ -100,7 +100,7 @@ class UtmForm
     fieldEl = document.createElement('input')
     fieldEl.type = "hidden"
     fieldEl.name = fieldName
-    fieldEl.value = if @_decodeURIs then decodeURIComponent(fieldValue) else fieldValue
+    fieldEl.value = if fieldValue then if @_decodeURIs then decodeURIComponent(fieldValue) else fieldValue else fieldValue
```

Comment:
`decodeURIComponent` of `null` returns `'null'` (a string with null value).

Reply:
Instead of the nested logic, how about this?

```javascript
value = fieldValue || ''
fieldEl.value = if @_decodeURIs then decodeURIComponent(value) else value
```

This makes it more readable and easy to understand.